app/v2/models/response/retrieveForeignPropertyAnnualSubmission/foreignFhlEea/ForeignFhlEeaAllowances.scala

```diff
@@ -16,14 +16,23 @@
 
 package v2.models.response.retrieveForeignPropertyAnnualSubmission.foreignFhlEea
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json.{JsPath, Json, OWrites, Reads}
+import play.api.libs.functional.syntax._
 
 case class ForeignFhlEeaAllowances(annualInvestmentAllowance: Option[BigDecimal],
                                    otherCapitalAllowance: Option[BigDecimal],
```

Comment:
fields dont match the spec. `propertyAllowance` should not be there and its `zeroEmissionsCarAllowance` should be